site/src/examples/slider/SingleFormField.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  FlexLayout,
+  FormField,
+  FormFieldLabel,
+  StackLayout,
+} from "@salt-ds/core";
+import { Slider } from "@salt-ds/lab";
+import type { ReactElement } from "react";
+
+export const SingleFormField = (): ReactElement => (
+  <StackLayout style={{ width: "400px" }}>
+    <FormField>
+      <FlexLayout gap={1} align="center">
+        <FormFieldLabel style={{ width: "max-content" }}>
+          Field Label
+        </FormFieldLabel>
+        <Slider
+          aria-label="single-form-field"
```

Comment:
This needs an a11y review, but currently, it feels broken as an example. This aria-label overrides the FF label announcement.

Reply:
Removed the aria-label override